R/kenwardroger.R

```diff
@@ -25,9 +25,9 @@ h_get_kr_comp <- function(tmb_data, theta) {
 
 #' Calculation of Kenward-Roger Degrees of Freedom for Multi-Dimensional Contrast
 #'
-#' @param object `mmrm` object
-#' @param contrast `matrix` contrast matrix
-#' @param linear `logical`\cr whether to use linear Kenward-Roger approximation
+#' @param object (`mmrm`)\cr object
+#' @param contrast (`matrix`)\cr contrast matrix
+#' @param linear (`logical`)\cr whether to use linear Kenward-Roger approximation
```

Comment:
```suggestion
#' @param object (`mmrm`)\cr object.
#' @param contrast (`matrix`)\cr contrast matrix.
#' @param linear (`logical`)\cr whether to use linear Kenward-Roger approximation.
```
another convention, we finish the argument description with full stop, thanks for doing that everywhere